SeaPublicWebsite/Views/Shared/_Layout.cshtml

```diff
@@ -101,11 +101,13 @@
                     </text>
         }))
 
-        @(RenderSection("BeforeMain", false))
 
-        <main class="govuk-main-wrapper" id="main-content" role="main">

```

Comment:
I realise that this is what the ticket says, however it's going directly against the GOV UK Design System guidance: https://design-system.service.gov.uk/components/back-link/

`Always place back links at the top of a page, before the <main> element. Placing them here means that the ‘Skip to main content’ link allows the user to skip all navigation links, including the back link.`

Reply:
reverted